UIforETW/Utility.cpp

```diff
@@ -547,9 +770,9 @@ std::wstring GetBuildTimeFromAddress(void* codeAddress)
 	// Print out the module information. The %.24s is necessary to trim
 	// the new line character off of the date string returned by asctime().
 	// _wasctime_s requires a 26-character buffer.
-	wchar_t ascTimeBuf[26];
+	wchar_t ascTimeBuf[26] = {0};

```

Comment:
I *figure* that if they do indeed null terminate, then the compiler will eliminate the zero initialization code during optimization, so it won't hurt. I should look at the actual generated code, though.